app/components/rdf-form-fields/rich-text-editor.js

```diff
@@ -0,0 +1,172 @@
+import SimpleInputFieldComponent from '@lblod/ember-submission-form-fields/components/rdf-input-fields/simple-value-input-field';
+
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+import { tracked } from '@glimmer/tracking';
+import { ProseParser, Schema, Selection } from '@lblod/ember-rdfa-editor';
+import {
+  block_rdfa,
+  doc,
+  hard_break,
+  horizontal_rule,
+  invisible_rdfa,
+  paragraph,
+  repaired_block,
+  text,
+} from '@lblod/ember-rdfa-editor/nodes';
+import { inline_rdfa } from '@lblod/ember-rdfa-editor/marks';
+import {
+  em,
+  strikethrough,
+  strong,
+  subscript,
+  superscript,
+  underline,
+} from '@lblod/ember-rdfa-editor/plugins/text-style';
+import {
+  tableKeymap,
+  tableNodes,
+  tablePlugin,
+} from '@lblod/ember-rdfa-editor/plugins/table';
+import { image } from '@lblod/ember-rdfa-editor/plugins/image';
+import { link, linkView } from '@lblod/ember-rdfa-editor/plugins/link';
+import { blockquote } from '@lblod/ember-rdfa-editor/plugins/blockquote';
+import { heading } from '@lblod/ember-rdfa-editor/plugins/heading';
+import { code_block } from '@lblod/ember-rdfa-editor/plugins/code';
+import {
+  bullet_list,
+  list_item,
+  ordered_list,
+} from '@lblod/ember-rdfa-editor/plugins/list';
+import {
+  bullet_list_input_rule,
+  ordered_list_input_rule,
+} from '@lblod/ember-rdfa-editor/plugins/list/input_rules';
+import { placeholder } from '@lblod/ember-rdfa-editor/plugins/placeholder';
+import { highlight } from '@lblod/ember-rdfa-editor/plugins/highlight/marks/highlight';
+import { color } from '@lblod/ember-rdfa-editor/plugins/color/marks/color';
+import { inputRules } from '@lblod/ember-rdfa-editor';
```

Comment:
Imported all of these as when removing some certain plugins stopped working correctly.